wagtail/contrib/styleguide/views.py

```diff
@@ -101,7 +101,10 @@ def media(self):
         )
 
 
-icon_id_pattern = re.compile(r"id=\"icon-([a-z0-9-]+)\"")
+# Allow no quotes, single quotes, and double quotes for the ID.
+# For simplicity and readability, we don't enforce the opening
+# and closing quotes to match.
+icon_id_pattern = re.compile(r"""id=["']?icon-([a-z0-9-]+)["']?""")
```

Comment:
I think we should drop the "no quotes" option, as it’s apparently invalid in XML – see the [AttValue](https://www.w3.org/TR/REC-xml/#NT-AttValue) parsing.

Reply:
Right, I forgot SVG is XML and not HTML!